.github/workflows/bump_packages.yml

```diff
@@ -112,16 +99,24 @@ jobs:
         title: "Update ${{ matrix.package_name }} to ${{ matrix.new_version }}"
         body: ''
         delete-branch: true
+
+  report:
+    name: Report failure to Discourse
+    needs:
+      - bump_plugin_deb
+      - bump_rpm
+    runs-on: ubuntu-latest
+    if: failure()
+    steps:
     - name: Send email to Discourse on failure
-      if: failure()
       uses: dawidd6/action-send-mail@v3
       with:
         server_address: community.theforeman.org
         server_port: 25
-        subject: "foreman-packaging update Debian packages Github Action failed"
+        subject: "foreman-packaging update packages Github Action failed"
         to: ci@community.theforeman.org
```

Comment:
Should we store this in a secret to avoid spam? It may already be out now, but hiding it in past revisions is probably safe enough.

Reply:
Would then also need hiding here? https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/lib/foreman_infra.groovy#L132

(We can, I just question the usefulness)

Reply:
I suppose. Perhaps I'm just a bit cautious with publicly visible email addresses.